backend/codesirius/codesirius/views.py

```
@@ -1,7 +1,14 @@
+from django.views.decorators.http import require_http_methods
 from rest_framework.exceptions import NotFound
+from rest_framework.response import Response
 from rest_framework.views import APIView
 
 
+@require_http_methods(["GET"])
+def health_check(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `health_check`

(ANN201)